include/exiv2/image.hpp

```diff
@@ -210,28 +210,28 @@ namespace Exiv2
                                bool bSwap, char c, int depth);
 
         /// @brief Is the host platform big Endian?
-        bool isBigEndianPlatform() const;
+        static bool isBigEndianPlatform();
```

Comment:
These changes are revealing that these `isXXX` and `byteSwap` functions should not be part of the `Image` class. These are generic utilities and not methods related to a specific image.

I might create a new issue documenting this discovery so that we can take care of that in the future.